docker/dockerfiles/Dockerfile

```diff
@@ -132,22 +133,29 @@ WORKDIR /tmp
 # en_US.UTF-8 locale not available by default
 RUN locale-gen en_US.UTF-8
 
-# install mysql
+# Install MySQL.
+#
+# We explicitly target MySQL 5.7, and because of that we also explicitly target
+# Ubuntu Bionic (18.x).
+#
+# We also explicitly install both mysql-client and mysql-server, so that we can
+# also explicitly define the versions of both; normally only the latter is
+# required since it depends on the former.
+#
+# Once we update to MySQL 8+, this can be simplified.
 RUN curl -sSLOJ https://dev.mysql.com/get/mysql-apt-config_0.8.22-1_all.deb && \
     echo "mysql-apt-config mysql-apt-config/select-server select mysql-5.7" | /usr/bin/debconf-set-selections && \
+    echo "mysql-apt-config mysql-apt-config/repo-codename select bionic" | /usr/bin/debconf-set-selections && \
     dpkg -i mysql-apt-config_0.8.22-1_all.deb || apt-get -fy install && \
     rm -rf mysql-apt-config_0.8.22-1_all.deb && \
     apt-get update && \
-    # warning: mysql-server debconf hangs forever if wget is already installed (!)
```

Comment:
Not only does this appear to no longer be true, but without `wget` already being installed, attempting to configure mysql with `dpkg -i mysql-apt-config_0.8.22-1_all.deb` will now fail *unless* `wget` is already installed.